src/Symfony/Bridge/Twig/NodeVisitor/Scope.php

```diff
@@ -40,7 +40,7 @@ public function enter()
      *
      * @return self|null
```

Comment:
```suggestion
```

Reply:
I'll delegate all similar changed to php-cs-fixer when it can do it.

Reply:
Okay. It's just that you've already removed the return annotations from other methods in this PR, but not here.

Reply:
yep, looks like something to fix forphp-cs-fixer when union types are used